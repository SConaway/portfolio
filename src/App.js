import React from 'react';
import Header from './Containers/Header/Header';
import Footer from './Containers/Footer/Footer';
import About from './Containers/About/About';
import Resume from './Containers/Resume/Resume';
import Contact from './Containers/Contact/Contact';
// import Testimonials from './Containers/Testimonials/Testimonials';
import Portfolio from './Containers/Portfolio/Portfolio';

const App = () => {
    return (
        <div className="App">
            <Header />
            <About />
            <Resume />
            <Portfolio />
            {/* <Testimonials /> */}
            <Contact />
            <Footer />
        </div>
    );
};

export default App;
