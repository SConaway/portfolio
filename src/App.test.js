import App from './App';

import renderer from 'react-test-renderer';

it('tree snapshot matches correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});
