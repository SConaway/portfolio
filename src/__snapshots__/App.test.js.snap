// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tree snapshot matches correctly 1`] = `
<div
  className="App"
>
  <header
    id="home"
  >
    <nav
      id="nav-wrap"
    >
      <a
        className="mobile-btn"
        href="#nav-wrap"
        title="Show navigation"
      >
        Show navigation
      </a>
      <a
        className="mobile-btn"
        href="#home"
        title="Hide navigation"
      >
        Hide navigation
      </a>
      <ul
        className="nav"
        id="nav"
      >
        <li
          className="current"
        >
          <a
            className="smoothscroll"
            href="#home"
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#about"
          >
            About
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#resume"
          >
            Resume
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#portfolio"
          >
            Works
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#contact"
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="row banner"
    >
      <div
        className="banner-text"
      >
        <h1
          className="responsive-headline"
        >
          Steven Conaway
        </h1>
        <h3>
          I'm an aspiring computer scientist and electrical engineer currently seeking opportunities in these fields.
        </h3>
        <hr />
        <ul
          className="social"
        >
          <li>
             
            <a
              href="https://linkedin.com/in/steven-conaway"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-linkedin fa-w-14 "
                data-icon="linkedin"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
             
          </li>
          <li>
             
            <a
              href="http://github.com/sconaway"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-github fa-w-16 "
                data-icon="github"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 496 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
             
          </li>
          <li>
             
            <a
              href="mailto://sjconaway48@gmail.com"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-envelope fa-w-16 "
                data-icon="envelope"
                data-prefix="fa"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
             
          </li>
        </ul>
      </div>
    </div>
    <p
      className="scrolldown"
    >
      <a
        className="smoothscroll"
        href="#about"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-angle-down fa-w-10 "
          data-icon="angle-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 320 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </a>
    </p>
  </header>
  <section
    id="about"
  >
    <div
      className="row"
    >
      <div
        className="three columns"
      >
        <img
          alt="Tim Baker Profile Pic"
          className="profile-pic"
          defer={true}
          src="profilePic.jpg"
        />
      </div>
      <div
        className="nine columns main-col"
      >
        <h2>
          About Me
        </h2>
        <p>
          While the high school I have attended does not offer classes in the ever-changing fields of technology, I have found a way to integrate these fields into my study: I have taken online courses in STEM fields independently and am the founding president of my school's VEX Robotics Club. Additionally, I have significant STEM experience outside of my school. I enjoy sharing my knowledge and experience with others in my community. Most recently, I was the Recipient of the Environmental Justice Angel Award at the East Los Angeles Community Youth Center's annual Gala in November 2019.
        </p>
        <div
          className="row"
        >
          <div
            className="columns contact-details"
          >
            <h2>
              Contact Details
            </h2>
            <p
              className="address"
            >
              <span>
                Steven Conaway
              </span>
              <br />
              <span>
                28874 Seco Canyon Road
                <br />
                Santa Clarita
                 
                CA
                ,
                 
                91390
              </span>
              <br />
              <span>
                (661) 236-5956
              </span>
              <br />
              <span>
                sjconaway48@gmail.com
              </span>
            </p>
          </div>
          <div
            className="columns download"
          >
            <p>
              <a
                className="button"
                href="https://stevenconaway.me/resume-2020-07-05.pdf"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-download fa-w-16 "
                  data-icon="download"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M216 0h80c13.3 0 24 10.7 24 24v168h87.7c17.8 0 26.7 21.5 14.1 34.1L269.7 378.3c-7.5 7.5-19.8 7.5-27.3 0L90.1 226.1c-12.6-12.6-3.7-34.1 14.1-34.1H192V24c0-13.3 10.7-24 24-24zm296 376v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h146.7l49 49c20.1 20.1 52.5 20.1 72.6 0l49-49H488c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                Download Resume
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="resume"
  >
    <div
      className="row education"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            Education
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <div
          className="row item"
        >
          <div
            className="twelve columns"
          >
            <div>
              <h3>
                Saint Monica Academy
              </h3>
              <p
                className="info"
              >
                High School Diploma
                 
                <span
                  className="bullet"
                >
                  •
                </span>
                <em
                  className="date"
                >
                  June 2021
                </em>
              </p>
              <p>
                Saint Monica Academy offers her students a classical education, teaching them critical thinking skills in addition to giving them an academically rigorous education. The teachers inspire students to excel at all things they attempt. While attending Saint Monica Academy's high school, I have received many honors. This past year, I was nominated to be president of our Honor Society. I am the founding president of our Robotics Club, which added an Engineering Track this year. As the project leader, I was responsible for organization of the project, planning meetings, and other administrative tasks.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row work"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            Work Experience
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <div>
          <h3>
            Mobile Application Developer
          </h3>
          <p
            className="info"
          >
            <span
              style={
                Object {
                  "fontWeight": "bold",
                  "textDecoration": "underline black solid",
                }
              }
            >
              We2Link
            </span>
            <span
              className="bullet"
            >
              •
            </span>
            <em
              className="date"
            >
              August 2019 – Present
            </em>
          </p>
          <p>
            Designing and programming a React Native mobile application for iOS and Android to help those with traumatic brain injuries. This project includes the Agile Development methodology and use of GitHub as a code repository and issue tracker, and Jira for project management.
          </p>
        </div>
        <div>
          <h3>
            Tech Instructor
          </h3>
          <p
            className="info"
          >
            <span
              style={
                Object {
                  "fontWeight": "bold",
                  "textDecoration": "underline black solid",
                }
              }
            >
              East Los Angeles Community Youth Center
            </span>
            <span
              className="bullet"
            >
              •
            </span>
            <em
              className="date"
            >
              June 2018 – March 2020
            </em>
          </p>
          <p>
            Taught a "How to Build your own Arduino-based Smart Car" class, "Introduction to Drones", and "LEGO Robotics" to underserved elementary- and middle-school students. Our Robotics team won second place in the First Annual Boyle Heights Robotics Competition. Also, developed the curriculum currently in use for the "How to Build your own Arduino-based Smart Car" class. 
          </p>
        </div>
        <div>
          <h3>
            Embedded Application Developer
          </h3>
          <p
            className="info"
          >
            <span
              style={
                Object {
                  "fontWeight": "bold",
                  "textDecoration": "underline black solid",
                }
              }
            >
              Unlocked Escape Rooms
            </span>
            <span
              className="bullet"
            >
              •
            </span>
            <em
              className="date"
            >
              November 2018 – May 2019
            </em>
          </p>
          <p>
            Designed and programmed Arduino-based props for escape rooms, such as RFID door locks. These props required the Arduino to interface with various hardware, such as RFID tags, electronic locks, and arrays of buttons.
          </p>
        </div>
        <div>
          <h3>
            Tutor
          </h3>
          <p
            className="info"
          >
            <span
              style={
                Object {
                  "fontWeight": "bold",
                  "textDecoration": "underline black solid",
                }
              }
            />
            <em
              className="date"
            >
              January 2018 – February 2020
            </em>
          </p>
          <p>
            Tutored middle- and high- school students from Saint Monica Academy in various subjects, including Latin, Math, and Java Programming.
          </p>
        </div>
      </div>
    </div>
    <div
      className="row skill"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            Skills
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <p>
          I've gained knowledge in many areas of Computer Science through my own projects and by the work done for others.
        </p>
        <div
          className="bars"
        >
          <ul
            className="skills"
          >
            <li>
              <span
                className="bar-expand react native"
                style={
                  Object {
                    "width": "100%",
                  }
                }
              />
              <em>
                React Native
              </em>
            </li>
            <li>
              <span
                className="bar-expand modern javascript"
                style={
                  Object {
                    "width": "100%",
                  }
                }
              />
              <em>
                Modern JavaScript
              </em>
            </li>
            <li>
              <span
                className="bar-expand html5"
                style={
                  Object {
                    "width": "90%",
                  }
                }
              />
              <em>
                HTML5
              </em>
            </li>
            <li>
              <span
                className="bar-expand arduino"
                style={
                  Object {
                    "width": "90%",
                  }
                }
              />
              <em>
                Arduino
              </em>
            </li>
            <li>
              <span
                className="bar-expand css"
                style={
                  Object {
                    "width": "80%",
                  }
                }
              />
              <em>
                CSS
              </em>
            </li>
            <li>
              <span
                className="bar-expand python"
                style={
                  Object {
                    "width": "70%",
                  }
                }
              />
              <em>
                Python
              </em>
            </li>
            <li>
              <span
                className="bar-expand git"
                style={
                  Object {
                    "width": "60%",
                  }
                }
              />
              <em>
                Git
              </em>
            </li>
            <li>
              <span
                className="bar-expand reactjs"
                style={
                  Object {
                    "width": "50%",
                  }
                }
              />
              <em>
                ReactJS
              </em>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </section>
  <section
    id="portfolio"
  >
    <div
      className="row"
    >
      <div
        className="twelve columns collapsed"
      >
        <h1>
          Check Out Some of My Works.
        </h1>
        <div
          className="bgrid-quarters s-bgrid-thirds cf"
          id="portfolio-wrapper"
        >
          <div
            className="columns portfolio-item"
          >
            <div
              className="item-wrap"
            >
              <a
                href="https://github.com/SConaway/portfolio"
                title="Resume Website"
              >
                <img
                  alt="Resume Website"
                  defer={true}
                  src="resume-website.jpeg"
                />
                <div
                  className="overlay"
                >
                  <div
                    className="portfolio-item-meta"
                  >
                    <h5>
                      Resume Website
                    </h5>
                    <p>
                      A resume website based on ReactJS.
                    </p>
                  </div>
                </div>
                <div
                  className="link-icon"
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-link fa-w-16 "
                    data-icon="link"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                </div>
              </a>
            </div>
          </div>
          <div
            className="columns portfolio-item"
          >
            <div
              className="item-wrap"
            >
              <a
                href="https://github.com/ELAScience/AQ-drone"
                title="CircuitPython-based Sensor Board"
              >
                <img
                  alt="CircuitPython-based Sensor Board"
                  defer={true}
                  src="sensors.jpeg"
                />
                <div
                  className="overlay"
                >
                  <div
                    className="portfolio-item-meta"
                  >
                    <h5>
                      CircuitPython-based Sensor Board
                    </h5>
                    <p>
                      A drone-mounted PCB with air quality sensors based on an Adafruit Feather running CircuitPython.
                    </p>
                  </div>
                </div>
                <div
                  className="link-icon"
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-link fa-w-16 "
                    data-icon="link"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="contact"
  >
    <div
      className="row section-head"
    >
      <div
        className="two columns header-col"
      >
        <h1>
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-envelope fa-w-16 "
            data-icon="envelope"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <span>
            Get In Touch
          </span>
        </h1>
      </div>
      <div
        className="ten columns"
      >
        <p
          className="lead"
        >
          Have a project for me? Think I'd be a good fit for your team? I'd love to hear from you. Send me an email using the form below.
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="eight columns"
      >
        <div
          id="message-warning"
        >
           Error
        </div>
        <div
          id="message-success"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-check fa-w-16 "
            data-icon="check"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          Your message was sent, thank you!
          <br />
        </div>
      </div>
      <aside
        className="four columns footer-widgets"
      >
        <div
          className="widget widget_contact"
        >
          <h4>
            Contact Information
          </h4>
          <h5>
            Phone
          </h5>
          (661) 236-5956
          <h5>
            Email
          </h5>
          sjconaway48@gmail.com
          <h5>
            Snail Mail
          </h5>
          <p
            className="address"
          >
            Steven Conaway
            <br />
            28874 Seco Canyon Road
             
            <br />
            Santa Clarita
            ,
             
            CA
             
            91390
            <br />
            <span>
              (661) 236-5956
            </span>
          </p>
        </div>
      </aside>
    </div>
  </section>
  <footer>
    <div
      className="row"
    >
      <div
        className="twelve columns"
      >
        <ul
          className="social-links"
        >
          <li>
            <a
              href="https://linkedin.com/in/steven-conaway"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-linkedin fa-w-14 "
                data-icon="linkedin"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
          </li>
          <li>
            <a
              href="http://github.com/sconaway"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-github fa-w-16 "
                data-icon="github"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 496 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
          </li>
          <li>
            <a
              href="mailto://sjconaway48@gmail.com"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-envelope fa-w-16 "
                data-icon="envelope"
                data-prefix="fa"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
          </li>
        </ul>
        <ul
          className="copyright"
        >
          <li>
            © Copyright 
            2020
             Steven Conaway
          </li>
          <li>
            <span>
              Original Design by
               
              <a
                href="http://www.styleshout.com/"
                title="Styleshout"
              >
                Styleshout
              </a>
            </span>
          </li>
          <li>
            <span>
              Converted to ReactJS by
               
              <a
                href="https://github.com/tbakerx/react-resume-template"
                title="Tim Baker"
              >
                Tim Baker
              </a>
            </span>
          </li>
        </ul>
      </div>
      <div
        id="go-top"
      >
        <a
          className="smoothscroll"
          href="#home"
          title="Back to Top"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-angle-up fa-w-10 "
            data-icon="angle-up"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M177 159.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 255.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 329.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </div>
    </div>
  </footer>
</div>
`;
